src/components/BookList.js

```diff
@@ -0,0 +1,43 @@
+export default function BookList() {
+  const booklist = [
+    {
+      title: 'The Hunger Games',
+      Author: 'Suzanne Collins',
+    },
+    {
+      title: 'Dune',
+      Author: 'Frank Herbert',
+    },
+    {
+      title: 'Capital in the Twenty-First Century',
+      Author: 'Suzanne Collins',
+    },
+  ];
+
+  return (
+    <div className="main-container">
+      <div className="bookList-container">
+        <div className="book-item">
+          {booklist.map((item) => (
+            <li key={item.id}>
+              {item.title}
+              <br />
+              {item.Author}
+              <br />
+              <br />
+            </li>
+          ))}
+        </div>
+        <hr />
+      </div>
+      <div className="form-container">
+        <h3 className="form-title">ADD NEW BOOK</h3>
+        <form className="addabook">
+          <input type="text" name="title" placeholder="Book title" />
+          <input type="text" name="author" placeholder="Author" />
+          <input type="submit" value="ADD BOOK" />
+        </form>
+      </div>
+    </div>
```

Comment:
- [ ] For this section, you are expected to have a separate component called AddBook or so, that is responsible for adding a book, then importing this component and rendering it in this file. Kindly move these codes into an AddBook.js component and use that component in space of these codes 👍🏽